Quick Start Guide (#283)
solidjs/solid-docs-next: content/quick-start.mdx

import { Aside } from "~/components/configurable/Aside";
import { CodeTabs } from "~/components/Tabs";

# Quick Start Guide

## Solid Browser Playgrounds

Experience Solid in your browser with our interactive playground throug visiting [playground.solidjs.com](https://playground.solidjs.com/).

> Additionally, we offer some Stackblitz starters, which provide a convenient web-based development environment. You can explore the [JavaScript](https://stackblitz.com/github/solidjs/templates/tree/master/js) or opt for our [Typescript](https://stackblitz.com/github/solidjs/templates/tree/master/ts) starter.

Reviewer: ```suggestion
Additionally, we offer some Stackblitz starters, which provide a convenient web-based development environment. You can explore the [JavaScript](https://stackblitz.com/github/solidjs/templates/tree/master/js) version or opt for our [Typescript](https://stackblitz.com/github/solidjs/templates/tree/master/ts) one.
```

## Creating a Solid Application

<Aside>
**Prerequisites:**
* *Familiarity with the command line.*
* *Install [Node.js](https://nodejs.org/en).*
</Aside>

Solid offers convenient project templates that can help kickstart your development. To get an application running, follow the steps below based on the language you prefer to use.

### For JavaScript Projects

1. Create application using a Solid template:

Solid provides a JavaScript template that's perfect to start with. To create your application structure, execute the following command in your terminal:

```bash
npx degit solidjs/templates/js my-app
```

3. Navigate to your application's directory

```bash
cd my-app
```

5. Install the necessary dependencies:
Using npm:
```bash
npm install
```
Alternatively, you can use yarn or pnpm:
```bash
yarn install
# or
pnpm install
```

7. Run the application:

```bash
npm run dev
```

This will start the development server. Now, you can open your browser and navigate to `localhost:3000` to see your application running.

> ### For TypeScript Projects

Reviewer: ```suggestion
### For TypeScript projects
```

If you prefer using TypeScript, Solid provides a TypeScript template as well. 

1. Create application using a Solid template:

> Solid provides a JavaScript template that's perfect to start with. To create your application structure, execute the following command in your terminal:

Reviewer: ```suggestion
Solid provides a great TypeScript starter template. To create your application structure, execute the following command in your terminal:
```

```bash
npx degit solidjs/templates/ts my-app
```

> 3. Navigate to your application's directory

Reviewer: ```suggestion
3. Navigate to your application's directory:
```

```bash
cd my-app
```

5. Install the necessary dependencies:
Using npm:
```bash
npm install
```
Alternatively, you can use yarn or pnpm:
```bash
yarn install
# or
pnpm install
```

7. Run the application:

> 

Reviewer: ```suggestion
```

```bash
npm run dev
```

> This will start the development server. Now, you can open your browser and navigate to `localhost:3000` to see your application running.

Reviewer: ```suggestion
This will start the development server. Now, open your browser and navigate to `localhost:3000` to see your application running.
```
